testing/e2e/config/config.go

```diff
@@ -156,12 +157,70 @@ type AdditionalService struct {
 	Replicas int `json:"replicas"`
 }
 
+// NethermindConfig holds specific configuration for Nethermind client
+type NethermindConfig struct {
+	// Specific settings for Nethermind
+	SyncConfig struct {
+		FastSync                  bool `json:"FastSync"`
+		DownloadBodiesInFastSync bool `json:"DownloadBodiesInFastSync"`
+		DownloadReceiptsInFastSync bool `json:"DownloadReceiptsInFastSync"`
+	} `json:"SyncConfig"`
+	
+	// Settings to improve finalization
+	ConsensusConfig struct {
+		ForceSealing bool `json:"ForceSealing"`
```

Comment:
`ForceSealing` is part of Aura namespace, pls check the correct usage. Refer https://docs.nethermind.io/fundamentals/configuration/#aura-forcesealing

I am curious why do we need it ? 
Another point to note is it is true by default.